server.go

```diff
@@ -388,10 +388,10 @@ func newServer(listenAddrs []net.Addr, chanDB *channeldb.DB,
 		ignorePeerTermination:   make(map[*peer]struct{}),
 		scheduledPeerConnection: make(map[string]func()),
 
-		peersByPub:                make(map[string]*peer),
-		inboundPeers:              make(map[string]*peer),
-		outboundPeers:             make(map[string]*peer),
-		peerConnectedListeners:    make(map[string][]chan<- lnpeer.Peer),
+		peersByPub:             make(map[string]*peer),
```

Comment:
what changed here?

Reply:
Was unintended formatting change. Fixed it.